Sources/PrimerSDK/Classes/User Interface/OAuth/PrimerLoadWebViewModel.swift

```diff
@@ -154,6 +154,7 @@ internal class ApayaLoadWebViewModel: PrimerLoadWebViewModelProtocol {
         let clientTokenService: ClientTokenServiceProtocol = DependencyContainer.resolve()
         clientTokenService.loadCheckoutConfig { [weak self] (error) in
             if let error = error {
+                Primer.shared.delegate?.checkoutFailed?(with: error)
```

Comment:
are we sure we only call this method once?

Reply:
I believe so, it is not called from `navigate`, nor the `router`.